pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -256,7 +286,9 @@ cdef class HashTable:
 # name, dtype, float_group, default_na_value
 dtypes = [('Float64', 'float64', True, 'np.nan'),
           ('UInt64', 'uint64', False, 0),
-          ('Int64', 'int64', False, 'NPY_NAT')]
+          ('Int64', 'int64', False, 'NPY_NAT'),
+          ('UInt32', 'uint32', False, 0),
+          ('Int32', 'int32', False, 0)]
```

Comment:
this migth be a problem not sure

Reply:
are you referring to the 0 here?

i looked at this and dont think its an issue, bc it is only a placeholder that never gets used

Reply:
This field  (`default_na_value`) is probably not really needed.

 Just set `na_value2` to 0 here: https://github.com/pandas-dev/pandas/blob/4cfa97a1808f31c8235d513a7aded32f87f8ae89/pandas/_libs/hashtable_class_helper.pxi.in#L410
and drop the whole else-branch here: https://github.com/pandas-dev/pandas/blob/4cfa97a1808f31c8235d513a7aded32f87f8ae89/pandas/_libs/hashtable_class_helper.pxi.in#L432-L433

Reply:
yeah that was my comment, but agree not germane for now.